src/solver/conv_multipass_wino3x3WrW.cpp

```diff
@@ -365,7 +364,7 @@ bool ConvWinograd3x3MultipassWrW<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>
 // HIP backend required for sending ptr (buffer + offset)
 // ROCBLAS for GEMM step
 
-#if(MIOPEN_BACKEND_HIP && MIOPEN_USE_ROCBLAS)
+#if(MIOPEN_BACKEND_HIP && (MIOPEN_USE_ROCBLAS || MIOPEN_USE_MIOPENTENSILE))
```

Comment:
[minor issue] IIRC miopentensile does not support FP16 and BF16. That means that if `MIOPEN_USE_ROCBLAS == false` and `MIOPEN_USE_MIOPENTENSILE == true`, then this IsApplicable() should return `false` for FP16 and BF16.

Reply:
done